server/routes/incidentDetails/incidentDetailsEdit.ts

```diff
@@ -59,37 +109,97 @@ export default class IncidentDetailsEditRoutes {
       data,
       reportingOfficer: reporter || '',
       submitButtonText: 'Save and continue',
+      exitIncidentDetailsEditPage: true,
+      exitButtonHref: `/place-the-prisoner-on-report/${prisonerNumber}/${id}`,
       reportPreviouslySubmitted: false,
+      inciteAnotherPrisonerInput,
+      assistAnotherPrisonerInput,
     })
   }
 
-  view = async (req: Request, res: Response): Promise<void> => this.renderView(req, res, {})
+  view = async (req: Request, res: Response): Promise<void> =>
```

Comment:
Hmm. why is this different to the way it was done in incidentDetails.ts?

I am struggling to see why you didn't reuse that code?! How different is it?